libs/flags/log_level_flag.go

```diff
@@ -25,7 +25,7 @@ type LogLevelFlag struct {
 
 func NewLogLevelFlag() LogLevelFlag {
 	return LogLevelFlag{
-		l: log.LevelDisabled,
+		l: log.LevelWarn,
```

Comment:
Nice. Overall I like this PR a lot, @denik! 

It's kind of hard to make sure that we don't show lots of undesired output suddenly, and the acceptance tests help a lot with that.